File: Backend/src/main/java/com/example/backend/dto/ContentBlockDto.java
package com.example.backend.dto;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class ContentBlockDto {
    private String type;      // "paragraph" или "image"
    private String text;
    private String style;     // "normal", "bold", "italic", "underlined"
    private Integer size;
    private String area;      // "left", "center", "right"
    private Integer imageId;
    private String url;
    private Integer width;
    private Integer height;
}

File: MyApplication/data/src/main/java/com/example/data/Hints/datasource/remote/dto/ContentBlockDto.kt
package com.example.data.Hints.datasource.remote.dto

data class ContentBlockDto(
    val type: String,        // "paragraph" или "image"
    val text: String? = null,
    val style: String? = null,  // "normal" "bold" "italic" "underlined"
    val size: Int? = null,
    val area: String? = null,   // "left" "center" "right"
    val imageId: Int? = null,
    val url: String? = null,
    val width: Int? = null,
    val height: Int? = null
)
